### ISSUE TITLE
pythonhosted.org deprecated in favor of readthedocs.org

### ISSUE THREAD

**Issue by sebalix** [OCA/odoorpc#16]
The button to upload the documentation has disappear on PyPI and pythonhosted.org has been deprecated, Read the Docs seems to be the new official channel to publish the documentation (https://github.com/pypa/pypi-legacy/issues/672#issuecomment-316125918).

One problem is the use of ``sphinx_ext`` in the documentation of OdooRPC, resulting with the following error when building the doc on readthedocs.org:

```python
Running Sphinx v1.6.5
making output directory...
loading translations [en]... done

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/odoorpc/envs/latest/local/lib/python2.7/site-packages/sphinx/cmdline.py", line 305, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/odoorpc/envs/latest/local/lib/python2.7/site-packages/sphinx/application.py", line 196, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/odoorpc/envs/latest/local/lib/python2.7/site-packages/sphinx/application.py", line 456, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/odoorpc/envs/latest/local/lib/python2.7/site-packages/sphinx/registry.py", line 199, in load_extension
    raise ExtensionError(__('Could not import extension %s') % extname, err)
ExtensionError: Could not import extension sphinx_ext.doctest_custom (exception: No module named sphinx_ext.doctest_custom)

Extension error:
Could not import extension sphinx_ext.doctest_custom (exception: No module named sphinx_ext.doctest_custom)
```

**Comment by pedrobaeza** [comment]
@hbrunn @guewen do you have experience with this, isn't it?

**Comment by sebalix** [comment]
Well, the problem is surely the package here https://github.com/OCA/odoorpc/tree/master/sphinx_ext .
It should work if we move its content under the ``odoorpc`` and configure Sphinx to import it from ``odoorpc.sphinx_ext.doctest_custom`` (https://github.com/OCA/odoorpc/blob/master/doc/source/conf.py#L34).

**Comment by sebalix** [comment]
Fixed in #17 , I just had to "read the doc"...
